mkdocs.yml

```diff
@@ -470,6 +471,7 @@ plugins:
       intelligence_center/integrations/opencti.md: cti/features/integrations/opencti.md
       intelligence_center/integrations/splunk.md: cti/features/integrations/splunk.md
       intelligence_center/integrations/thehive.md: cti/features/integrations/thehive.md
+      intelligence_center/integrations/paloalto_xsoar.md: cti/features/integrations/paloalto_xsoar.md
```

Comment:
This redirection isn’t necessary. It mainly redirect old URLs (before reorganization) to new content (and `intelligence_center/integrations/paloalto_xsoar.md` document never existed).

Reply:
Removed